src/pages/signin/SignInPageLayout/BackgroundImage/index.ios.js

```diff
@@ -0,0 +1,38 @@
+import {Image} from 'expo-image';
+import PropTypes from 'prop-types';
+import React from 'react';
+import DesktopBackgroundImage from '@assets/images/home-background--desktop.svg';
+import MobileBackgroundImage from '@assets/images/home-background--mobile-new.svg';
+import useThemeStyles from '@hooks/useThemeStyles';
+import defaultPropTypes from './propTypes';
+
+const defaultProps = {
+    isSmallScreen: false,
+};
+
+const propTypes = {
+    /** Is the window width narrow, like on a mobile device */
+    isSmallScreen: PropTypes.bool,
+
+    ...defaultPropTypes,
+};
+function BackgroundImage(props) {
+    const styles = useThemeStyles();
+    return props.isSmallScreen ? (
+        <Image
+            source={MobileBackgroundImage}
```

Comment:
Let's update this to:

- Just make the source conditional
- Memoize the source
- Use `StyleUtils.getWidthStyle` instead of an inline style. I know these are basically the same but it's just our code style

```js
const src = useMemo(() => props.isSmallScreen ? MobileBackgroundImage : DesktopBackgroundImage, [props.isSmallScreen]);

return <Image source={src} style={[styles.signInBackground, StyleUtils.getWidthStyle(props.width)]} />
```